### PR TITLE
create modularity

### PR DESCRIPTION
**Pull request by DirectorSloan**
have it in master to check it with ansible

### CODE REVIEW

**Comment by wookietreiber** [comment]
Sorry about the delay @DirectorSloan, doing the review now.

File: tasks/install_suse.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - monitoring-plugins-swap
+      - monitoring-plugins-time
+      - monitoring-plugins-ntp_peer
+      - monitoring-plugins-ntp_time
+      - monitoring-plugins-wl-files
+      - monitoring-plugins-wl-dnsname
+  register: nrpe_installed
+  until: nrpe_installed is succeeded
+
+...
```
Comment (wookietreiber) [inline tasks/install_suse.yml:21]: Please add final newline here.

File: tasks/install_redhat.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- name: install nrpe for Red Hat Family
+  yum:
+    name:
+      - nrpe
+      - nagios-plugins-load
+      - nagios-plugins-procs
+      - nagios-plugins-users
+      - nagios-plugins-disk
+  register: nrpe_installed
+  until: nrpe_installed is succeeded
```
Comment (wookietreiber) [inline tasks/install_redhat.yml:12]: Please use `__nrpe_installed` for internal variables.

File: tasks/install_suse.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      - monitoring-plugins-disk
+      - monitoring-plugins-load
+      - monitoring-plugins-swap
+      - monitoring-plugins-time
+      - monitoring-plugins-ntp_peer
+      - monitoring-plugins-ntp_time
+      - monitoring-plugins-wl-files
+      - monitoring-plugins-wl-dnsname
+  register: nrpe_installed
+  until: nrpe_installed is succeeded
```
Comment (wookietreiber) [inline tasks/install_suse.yml:19]: Please use `__nrpe_installed` for internal variables.

**Review by wookietreiber — CHANGES_REQUESTED** [review]

**Review by wookietreiber — CHANGES_REQUESTED** [review]
Since this PR is basically about adding SUSE to the supported distros, please add the OpenSUSE versions you tested with to the `meta/main.yml` platforms list. You can find the versions supported by Ansible Galaxy [here](https://galaxy.ansible.com/api/v1/platforms/?search=suse).

**Comment by DirectorSloan** [comment]
do much delay, closing pr

**Pull request closed without merging**